Automattic/wp-calypso
# Plans: Optimize Fresh Start sale banner file sizes

#### Changes proposed in this Pull Request

Relates to `1196108640073826-as-1199527862646257`.

* Replace the existing "Fresh Start" sale banner image files with optimized versions that appear the same but that consume 70% less bandwidth.

#### Testing instructions



* Open Calypso Blue, Green, _and_ Jetpack Connect in your testing environment.
* Ensure that if the `jetpackNewPurchaseIntroPricing` test is active, you are assigned to the `withoutIntroPricing` variant.
* Verify that banners appear to be the same as in production for all screen layouts (HiDPI+LoDPI, mobile+desktop).

#### Reference screenshots

<img height="200" alt="image" src="https://user-images.githubusercontent.com/670067/108367132-c18e8580-71be-11eb-819f-b8ee84b6b60c.png"> <img height="200" alt="image" src="https://user-images.githubusercontent.com/670067/108367209-d23efb80-71be-11eb-89a5-5f84bc9c61a1.png">
